src/libraries/System.Runtime.Extensions/tests/System/Numerics/BitOperationsTests.cs

```diff
@@ -388,5 +388,32 @@ public static void BitOps_RotateRight_ulong()
             Assert.Equal(value, BitOperations.RotateRight(value, int.MinValue)); // % 64 = 0
             Assert.Equal(BitOperations.RotateLeft(value, 63), BitOperations.RotateRight(value, int.MaxValue)); // % 64 = 63
         }
+
+        [Theory]
+        [InlineData(0u, 0u)]
+        [InlineData(1u, 1u)]
+        [InlineData(2u, 2u)]
+        [InlineData(0x7FFF_FFFFu, 0x8000_0000u)]
+        [InlineData(0x8000_0000u, 0x8000_0000u)]
+        [InlineData(0xFFFF_FFFFu, 0)]
```

Comment:
Can we also add a test for `8000_0001`? and a couple values between `2` and `int.MaxValue`?

Just a couple random values rather than explicitly just the edge cases would be useful I think

Reply:
I got them literally using `Random`